libs/perun/pipes/src/lib/member-status-tooltip.pipe.ts

```diff
@@ -26,16 +26,21 @@ export class MemberStatusTooltipPipe implements PipeTransform {
       }`;
     } else {
       if (member.memberAttributes) {
-        res = `Status: ${parseMemberStatus(member.status, member.groupStatus)}
-             Vo status: ${parseMemberStatus(member.status)}, Expiration: ${
-          (memberExpiration?.value as string) ?? 'never'
-        }`;
+        res = `Status: ${parseMemberStatus(member.status, member.groupStatus)},
+             Vo status: ${parseMemberStatus(member.status)}, 
+             Expiration: ${(memberExpiration?.value as string) ?? 'never'}`;
       } else {
         // we can't display correct expiration without attributes, but we can still display statuses
-        res = `Status: ${parseMemberStatus(member.status, member.groupStatus)}
-             Vo status: ${parseMemberStatus(member.status)}`;
+        res = `Status: ${parseMemberStatus(member.status, member.groupStatus)},
+            Vo status: ${parseMemberStatus(member.status)}`;
       }
     }
+    if (isMemberIndirect(member)) {
+      res = `${res}, Members from member organizations cannot be directly altered!`;
+    }
+    if (membersGroup) {
+      res = `${res}, Members from 'members' group cannot be directly altered!`;
```

Comment:
Move hard-coded text descriptions etc. to en.json (to make future localization easier)